app/models/azure_engine.rb

```diff
@@ -97,8 +94,8 @@ def self.get_file_list(container, prefix)
 
   def self.save_file(container, key, destination)
     bs = Azure::Blob::BlobService.new
-    blob, content = bs.get_blob(container, key)
-    File.open(destination, "wb") {|f| f.write(content)}
+    _blob, content = bs.get_blob(container, key)
+    File.open(destination, 'wb') { |f| f.write(content) }
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'f' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)